.github/workflows/build.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ secrets.CR_USER }} --password-stdin
         docker push ppiper/node-browsers:${{ matrix.base-image-tag }}
         docker push ghcr.io/sap/ppiper-node-browsers:${{ matrix.base-image-tag }}
-        if [ "${{ matrix.base-image-tag }}" == 20-bookworm ]; then
+        if [ "${{ matrix.base-image-tag }}" == 22-bookworm ]; then
```

Comment:
@kaylinche Question: should we already tag node 22 as latest, or wait for the official switch as Active LTS before tagging it latest?

Reply:
From my side it is fine to tag 22 already as latest.